### DISCUSSION TITLE
LHR card(s) not detected with TON + ETH mining and shifting ETH vs TON priority

### DISCUSSION THREAD

**Discussion by Hawkica** [Lolliedieb/lolMiner-releases#1203] [Q&A]
Each time I start the dual mining there is a shift in terms of some cards not being detected as LHR and are achieve a much lower speed for ETH. I have a fully 3060 ti LHR rig and am using RaveOS.
**Example:**
_I start the mining and after some time my speeds look like this:_
ETH: 44 MH/s 31 MH/s 42 MH/s 45 MH/s...
TON: 700 MH/s 1400 MH/s 800 MH/s 600 MH/s...
For card 1 (31 MH/s) I get the message - It seems this card is not LHR. Sorry for bothering you.

_Next time I start the miner this is what happens:_
ETH: 29 MH/s 44 MH/s 34 MH/s 45 MH/s...
TON: 1500 MH/s 600 MH/s 1300 MH/s 700 MH/s...
Next time it's again different and I can't seem to reach this ideal state where ETH is above 43, 44 MH/s and the rest is covered by TON. Any idea here?

**Comment by Hawkica** [comment]
Am I really the only one facing this problem?

**Reply by atnzdhy** [reply]
嗯

**Comment by nraudigy2** [comment]
I get this all the time. LHR cards are detecting as Non-LHR or vice versa. Developer needs to add a command for us to identify the cards as LHR cards and not let the miner auto detect because auto detect is very bad.

**Comment by matheuskshn** [comment]
Yes I am also facing the same problem.

**Reply by matheuskshn** [reply]
I managed to solve the problem by specifying a value for --lhrtune, in my case as before in the logs the automatic configuration configured the parameter --lhrtune 2.4, I used this same value in the miner configuration and it managed to detect my 3070ti LHR, see my settings:

`--algo ETHASH --pool (POOL:FLEXPOOL-BR-SSL)⁣ --user (WALLET:ETH-LINO-BIN)⁣ --tls 1 --dualmode TONDUAL --ton-mode 2 --dualpool (POOL:TON-TONPOOL)⁣ --dualuser (WALLET:TON-LINO-TON_ORG)⁣ --worker (WORKER)⁣ --cclk 1275 --lhrtune 2.4​`

The options in parentheses is because I use minestat on windows.

**Reply by Hawkica** [reply]
Nope, neither locking the core clock nor specifying the lhrtune solved this for me.
